signal_boxes.mss

```diff
@@ -13,15 +13,15 @@
   line-width: 1;
 }
 
-#signal_boxes_point[zoom>=13] {
+#signal_boxes_point[zoom>=11] {
   /* render a circle, therefore marker-file is not set */
   marker-fill: @signal_box_color;
   marker-line-width: 1;
   marker-line-color: @signal_box_casing_color;
   marker-width: @signal_box_size;
   marker-height: @signal_box_size;
   [zoom>=15] {
-    marker-width: @signal_box_size_z15;
+    marker-width: @signal_box_size_z16;
```

Comment:
Carto error:

`Error: signal_boxes.mss:24:18 variable @signal_box_size_z16 is undefined`